bsb/morphologies/__init__.py

```diff
@@ -989,6 +1016,19 @@ class Branch:
     """
 
     def __init__(self, points, radii, labels=None, properties=None, children=None):
+        """
+        :param points: Array of 3D coordinates defining the point of the branch
+        :type points: list | :class:`numpy.ndarray`
+        :param radii: Array of radius associated to each point
+        :type radii: list|:class:`numpy.ndarray`
+        :param labels: Array of label to associate to each point
+        :type labels: EncodedLabels | list[str] | set | :class:`numpy.ndarray`
```

Comment:
```suggestion
        :type labels: EncodedLabels | List[str] | set | numpy.ndarray
```

To be python 3.8 compatible any type index needs to come from `typings`, so `typings.List`, so `List`.